xml/System.Text.Json/JsonSerializer.xml

```diff
@@ -391,13 +531,13 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" />
       </Parameters>
       <Docs>
-        <param name="utf8Json">To be added.</param>
-        <param name="value">To be added.</param>
-        <param name="inputType">To be added.</param>
-        <param name="options">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="utf8Json">The UTF-8 <see cref="T:System.IO.Stream" /> to write to.</param>
```

Comment:
Bullet No. 2 of [parameters guidelines](https://github.com/dotnet/dotnet-api-docs/wiki/Parameters) specifies that you should not include a link to the parameter type because that is automatically added by the build.
```suggestion
        <param name="utf8Json">The UTF-8 stream to write to.</param>
```